### PR TITLE
test: rewrite and rename an interception test

### PR DESCRIPTION
**Pull request by OrKoN**
to prevent opaque origin response blocking and clarify what the test does.

### CODE REVIEW

**Comment by OrKoN** [comment]
cc @juliandescottes

File: test/src/requestinterception-experimental.spec.ts [hunk 1]
```diff
@@ -603,23 +603,21 @@ describe('cooperative request interception', function () {
...
+    it('should work with missing stylesheets', async () => {
       const {page, server} = await getTestState();
 
       // The requestWillBeSent will report URL as-is, whereas interception will
       // report encoded URL for stylesheet. @see crbug.com/759388
       await page.setRequestInterception(true);
       const requests: HTTPRequest[] = [];
       page.on('request', request => {
-        void request.continue({}, 0);
+        void request.continue(undefined, 0);
```
Comment (Lightning00Blade) [inline test/src/requestinterception-experimental.spec.ts]: Why is this change necessary?

**Review by Lightning00Blade — APPROVED** [review]

File: test/src/requestinterception-experimental.spec.ts (hunk 1 above)
Comment (OrKoN) [reply test/src/requestinterception-experimental.spec.ts]: not necessary
